pkg/metrics3/client.go

```diff
@@ -0,0 +1,228 @@
+package metrics3
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type AnodotToken struct {
```

Comment:
```
type TokenType string

const (
	Api    TokenType = "api"
	Bearer           = "bearer"
)
```
What about changing to such kind of "enum" here?

Reply:
also, I think we need to use for 3.0 client 2 parameters
1. Data Collection Token - to send metrics
2. Access token to get `bearer` token which is valid for 24 hours.
https://docs.anodot.com/#basic-authentication

So in general, refreshing and getting a `bearer` token should be transparent for the user that is using this library.

Reply:
Renamed